packages/@webex/plugin-meetings/src/locus-info/parser.ts

```diff
@@ -288,7 +289,13 @@ export default class Parser {
         break;
 
       case LT:
-        if (extract(Parser.compareSequence(incoming.baseSequence, incoming.sequence)) === EQ) {
+        if (incoming.url !== current.url) {
```

Comment:
this will work, but conceptually it feels like this check should be done before the `switch()` statement as it really doesn't make any sense to be doing any logic based on the comparison if the delta event is for different url, so I think it would be better to move this up before line 285